### ISSUE TITLE
Unable to execute API command createLBStickine due to missing parameter methodname

### ISSUE THREAD

**Issue by tuanhoangth1603** [apache/cloudstack#7413]
Hello,
I am using Cloudstack 4.17.2.0
After created a load balancer rule, I click on Configure, choose a sticky method, enter Sticky name and click OK but it shows an error: 
 ```
Request failed. (431)
Unable to execute API command createLBStickine due to missing parameter methodname
```
I also tried all method: AppCookie, Lbcookie, SourceBased and got the same result.
![image](https://user-images.githubusercontent.com/38004836/230348766-aa2a7bce-72fa-430d-a143-47bcf8befbb5.png)

Please help ! Thanks a lots

**Comment by weizhouapache** [comment]
@tuanhoangth1603 
I remember this has been fixed in #6571 
maybe we need to backport to 4.17

**Comment by tuanhoangth1603** [comment]
Cloud we configure sticky via API ? And how to ?

**Comment by weizhouapache** [comment]
@tuanhoangth1603 
yes, you can use cloudmonkey
https://github.com/apache/cloudstack-cloudmonkey/releases

**Comment by weizhouapache** [comment]
this has been fixed in #6571 by commit https://github.com/apache/cloudstack/pull/6571/commits/b4d3deaddf6725a0553a50470bc41dcc951fab38
